src/android/RemoteInjectionPlugin.java

```diff
@@ -217,104 +233,98 @@ private static class RequestLifecycle {
         }
 
         boolean isLoading() {
-            return task != null;
+            return prompt != null;
+        }
+
+        public boolean isErrorOccured() {
```

Comment:
他関数と同様 package private で良さそうに思います（setter も同様です）。  
```suggestion
        boolean isErrorOccured() {
```

Reply:
こちら setter の public が削除されていないように見えますが、削除できなさそうでしたか。

Reply:
削除しました。

Reply:
対応しました